pkg/executor/executortype/poolmgr/gpm.go

```diff
@@ -166,11 +166,19 @@ func (gpm *GenericPoolManager) GetFuncSvc(ctx context.Context, fn *fv1.Function)
 }
 
 func (gpm *GenericPoolManager) GetFuncSvcFromCache(fn *fv1.Function) (*fscache.FuncSvc, error) {
-	return gpm.fsCache.GetByFunction(&fn.ObjectMeta)
+	return gpm.fsCache.GetConnFunction(&fn.ObjectMeta)
```

Comment:
What does `GetConnFunction` mean - or can we use a better name, please?

Reply:
Done